public/js/qr/history.js

```
@@ -0,0 +1,95 @@
+            return;
+        }
+
+        const user = firebase.auth().currentUser;
+        if (!user) {
+            console.log('No user logged in');
+            this.quickLinksGrid.innerHTML = '<p class="text-muted">Please log in to see your links</p>';
+            return;
+        }
+
+        const token = await user.getIdToken();
+        const response = await fetch('/api/user/links', {
```

ESLint

[error] 'fetch' is not defined.

(no-undef)